lib/tendrl/job.rb

```diff
@@ -25,11 +25,12 @@ def default_payload
       }
     end
 
-    def create(body, node_ids)
+    def create(body, routing = {})
```

Comment:
Something like `create(body, node_ids:[], tags:[])` could work and prevent checks for those later in the method.